packages/factom-did-provider/src/__tests__/factom-did-provider.test.ts

```diff
@@ -70,11 +70,6 @@ describe('@sphereon/factom-did-provider', () => {
   } as IRequiredContext*/
 
   it('should create identifier', () => {
-    const restResponse = {
```

Comment:
why is being being changed here?

Reply:
Because as per the updated tsconfig. This was giving error. This is unused any way.